lib/travis/api/v3/billing_client.rb

```diff
@@ -17,6 +17,12 @@ def get_subscription(id)
       handle_errors_and_respond(response)
     end
 
+    def get_invoices_for_subscription(id)
```

Comment:
we are a bit inconsistent with the `id`s here, sometimes it is `subscription_id` and sometimes just `id`. :woman_shrugging:

Reply:
I followed the previous method https://github.com/travis-ci/travis-api/blob/621bf5a3eab0ea3c44e154c1750e5e539f067bcf/lib/travis/api/v3/billing_client.rb#L15 
I think it's alright, since the name is very descriptive `get_subscription(id)` we know the id if for  a subscription.
Same for `get_invoices_for_subscription(id)`